tests/utils.ts

```diff
@@ -150,12 +153,33 @@ export async function checkRoutesXSol(
           user,
           feeAccounts,
         });
+        await sleep(Math.random() * MAX_RANDOM_JITTER_MS);
         await trySimulateExchangeReturnFirstTx(
           unstake,
           exchangeReturn,
           user,
           routeLabel,
         );
+        // try with a generated keypair too if default is by seed, to make sure that works
+        if (
+          !isXSolRouteJupDirect(route) &&
+          !route.withdrawStake.withdrawStakePool.mustUseKeypairForSplitStake
+        ) {
+          const exchangeReturnKp = await unstake.exchangeXSol({
+            route,
+            srcTokenAccount: xSolTokenAcc,
+            user,
+            feeAccounts,
+            newStakeAccount: Keypair.generate(),
+          });
+          await sleep(Math.random() * MAX_RANDOM_JITTER_MS);
```

Comment:
Is sleep less than a second always enough?

Reply:
yeah it looks like it actly, changing to 1s